src/core/public/core_app/status/components/metric_tiles.test.tsx

```diff
@@ -55,4 +67,9 @@ describe('MetricTile', () => {
     const component = shallow(<MetricTile metric={timeMetric} />);
     expect(component).toMatchSnapshot();
   });
+
+  it('correctly displays a metric with metadata', () => {
+    const component = shallow(<MetricTile metric={metricWithMeta} />);
+    expect(component).toMatchSnapshot();
+  });
```

Comment:
(thinking out loud, unrelated to the PR, don't mind me) snapshot testing was without any doubt the worse thing React did bring to the javascript ecosystem. This thing is a plague.

Reply:
> This thing is a plague.

Ditto to that.

Reply:
💯